test/runtests.jl

```diff
@@ -84,3 +84,9 @@ d = test_get_dimension(f);
 @test d.width == 576
 
 end
+
+@testset "error handling" begin
+
+@test_throws ErrorException Rsvg.handle_new_from_data("not a valid data input")
```

Comment:
Can this test be more specific?  `ErrorException` is what is thrown when a function is not found in a library, so that this tests would pass also without the fix